sway/input/cursor.c

```diff
@@ -1119,6 +1120,14 @@ static void handle_tool_tip(struct wl_listener *listener, void *data) {
 	transaction_commit_dirty();
 }
 
+static void handle_tool_proximity(struct wl_listener *listener, void *data) {
+	wlr_log(WLR_DEBUG, "proximity event");
```

Comment:
Is it possible to remove this? It happens often and will clutter debug logs?

Reply:
Whoops, forgot to remove that after testing